GMAO_pFIO/tests/pfio_performance.F90

```diff
@@ -64,29 +59,29 @@ subroutine process_command_line(options, rc)
          select case (argument)
          case ('-nc', '--npes_client')
             buffer = get_next_argument()
-            _ASSERT(buffer /= '-')
+            _ASSERT(buffer /= '-', "too many -")
             read(buffer,*) options%npes_client
          case ('-nso', '--nodes_oserver')
             buffer = get_next_argument()
-            _ASSERT(buffer /= '-')
+            _ASSERT(buffer /= '-',"too many -")
             read(buffer,*) options%nodes_oserver
          case ('-ncol', '--num_collection')
             buffer = get_next_argument()
-            _ASSERT(buffer /= '-')
+            _ASSERT(buffer /= '-',"too many -")
             read(buffer,*) options%num_collection
          case ('-f1', '--file_1')
             options%file_1 = get_next_argument()
-            _ASSERT(options%file_1(1:1) /= '-')
+            _ASSERT(options%file_1(1:1) /= '-',"too many -")
          case ('-f2', '--file_2')
             options%file_2 = get_next_argument()
-            _ASSERT(options%file_2(1:1) /= '-')
+            _ASSERT(options%file_2(1:1) /= '-',"too many -")
          case ('-v', '--var')
             buffer = get_next_argument()
-            _ASSERT(buffer(1:1) /= '-')
+            _ASSERT(buffer(1:1) /= '-',"too many -")
```

Comment:
Shouldn't these use FLAP?  Or is that not worth the trouble?

Reply:
I wouldn't put FLAP in this. Maybe fArgParse?

Reply:
I agree FLAP is too much.   Is fArgParse already in GEOS?